src/views/world/view_nearby.cpp

```diff
@@ -42,6 +42,10 @@ namespace big
 		components::command_checkbox<"highalert">();
 		ImGui::SameLine(140.f);
 		components::command_checkbox<"pedrush">();
+		components::command_checkbox<"autodisarm">();
```

Comment:
Personally I would put this inside the modal so the modal contains all the items related to the feature

Reply:
The rest of similar elements are designed in this way. Where the global enable variable is readily available, and its subordinates within a modal. Although this was my idea, I am open to suggestions since it is not the prettiest look.